example/server/mock_banking_rails.py

```diff
@@ -0,0 +1,61 @@
+"""
+This module contains a fake banking rails interface to use for the purposes of
+this example anchor server. In reality, the anchor should use whatever interface
+connects them with their real bank account.
+"""
+from uuid import uuid4
+from decimal import Decimal
+from polaris.models import Transaction
+
+
+class RailsAccount:
+    def __init__(self, account_id: str):
+        self.id = account_id
+
+
+class RailsTransaction:
+    def __init__(self, to_account: RailsAccount, amount: Decimal, memo: str):
+        self.id = str(uuid4())
+        self.to_account = to_account
+        self.from_account = RailsAccount(str(uuid4()))
+        self.amount = amount
+        self.status = "complete"
+        self.memo = memo
+
+
+class RailsClient:
```

Comment:
Can we call this MyBankAdapter or something? RailsClient doesn't seem very clear, it should be an adapter between polaris and the banking api right?

Reply:
Sure